cvat/settings/base.py

```diff
@@ -417,4 +417,8 @@ def generate_ssh_keys():
 LOCAL_LOAD_MAX_FILES_SIZE = 512 * 1024 * 1024  # 512 MB
 
 DATUMARO_PATH = os.path.join(BASE_DIR, 'datumaro')
-sys.path.append(DATUMARO_PATH)
\ No newline at end of file
+sys.path.append(DATUMARO_PATH)
+
+RESTRICTIONS = {
```

Comment:
Is it the place where agreements should be defined? If so, maybe some kind of plug-in folder (maybe with auto detection) could be better?

Reply:
Yes it's one of the possibles solution. I used the approach that we have used many times before.